MapboxNavigation/Constants.swift

```diff
@@ -6,7 +6,7 @@ public let RouteControllerProgressDidChangeNotificationProgressKey = MBRouteCont
 public let RouteControllerProgressDidChangeNotificationLocationKey = MBRouteControllerProgressDidChangeNotificationLocationKey
 public let RouteControllerProgressDidChangeNotificationSecondsRemainingOnStepKey = MBRouteControllerProgressDidChangeNotificationSecondsRemainingOnStepKey
 
-public let RouteControllerAlertLevelDidChangeNotificationRouteProgressKey = MBRouteControllerAlertLevelDidChangeNotificationRouteProgressKey
+public let RouteControllerDidChangeNotificationRouteProgress = MBRouteControllerDidChangeNotificationRouteProgress
```

Comment:
Nit: “Key” should be part of the _name_ of the constant, just not its string value.